### PR TITLE
[docs] Describe full page screenshots

### PR DESCRIPTION
**Pull request by VasilyStrelyaev**

### CODE REVIEW

**Comment by sharathgrao** [comment]
Hello Team - I'm not seeing the fullpage: true screenshot option in the latest 1.5.0 build. Can you please check?

**Comment by AndreyBelym** [comment]
Hi @sharathgrao, it's not released yet. It will be available in `testcafe@1.6.0`. If you want to test this feature, you can install `testcafe@1.6.0-rc.1`. Please note, that `testcafe@1.6.0-rc.1` is not a stable build, and it's better to avoid installing it in production environments.

File: docs/articles/documentation/using-testcafe/configuration-file.md [hunk 1]
```diff
@@ -203,53 +209,167 @@ You can use multiple reporters, but note that only one reporter can write to `st
...
+
+**Obsolete.** Specifies a custom pattern to compose screenshot files' relative path and name.
+
+```json
+{
+    "screenshotPathPattern": "${DATE}_${TIME}/test-${TEST_INDEX}/${USERAGENT}/${FILE_INDEX}.png"
+}
+```
+
+This property will be deprecated in the future versions.
```
Comment (dirk-pieterse) [inline docs/articles/documentation/using-testcafe/configuration-file.md]: Why do you add this sentence?

File: docs/articles/documentation/using-testcafe/configuration-file.md [hunk 2]
```diff
@@ -203,53 +209,167 @@ You can use multiple reporters, but note that only one reporter can write to `st
...
+
+Prevents TestCafe from taking screenshots.
+
+```json
+{
+    "disableScreenshots": true
+}
+```
+
+When this property is specified, screenshots are not taken whenever a test fails or a [screenshot action](../test-api/actions/take-screenshot.md) is executed.
```
Comment (dirk-pieterse) [inline docs/articles/documentation/using-testcafe/configuration-file.md]: When this property is specified, screenshots are not taken when a test fails or a [screenshot action](../test-api/actions/take-screenshot.md) is executed.

**Pull request merged**